ckanext/ogdchcommands/commands.py

```diff
@@ -17,6 +17,14 @@
 ==================
 {} resources in status 'deleted' have been deleted."""
 
+msg_package_extra_cleanup_dryrun = """\npackage extra cleanup for key '{1}':\n\n
+There are {0} package extras with key '{1}'.
+If you want to delete them, run this command
+again without the option --dryrun!\n"""
```

Comment:
I guess here it should be the other way around:
with option `--dryrun` and not without, as it is written

Reply:
This is the message the user gets when he runs the command with the dryrun option. He is hinted to run the command again and skip that option in order to do the real changes. So I think this is correct as it is. Unless you object I will leave it as it is.

This is the message in action when the command is run:

```
>docker-compose exec ckan /usr/local/bin/ckan-paster --plugin=ckanext-ogdchcommands ogdch cleanup_extras publisher --dryrun -c /etc/ckan/production.ini 

package extra cleanup for key 'publisher':

There are 11 package extras with key 'publisher'.
If you want to delete them, run this command
again without the option --dryrun!
```